packages/grafana-toolkit/src/cli/tasks/component.create.ts

```diff
@@ -0,0 +1,73 @@
+import { Task, TaskRunner } from './task';
+import fs from 'fs';
+import _ from 'lodash';
+import { prompt } from 'inquirer';
+import { pascalCase } from '../utils/pascalCase';
+import { promptConfirm, promptInput } from '../utils/prompt';
+import { componentTpl, docsTpl, storyTpl, testTpl } from '../templates';
+
+interface Details {
+  name?: string;
+  hasStory: boolean;
+  group?: string;
+  isStoryPublic: boolean;
+  hasTests: boolean;
+}
+
+interface GeneratorOptions {
+  details: Details;
+  path: string;
+}
+
+type ComponentGenerator = (options: GeneratorOptions) => Promise<any>;
+
+export const promptDetails = () => {
+  return prompt<Details>([
+    promptInput('name', 'Component name', true),
+    promptConfirm('hasTests', "Generate component's test file?"),
+    promptConfirm('hasStory', "Generate component's story file?"),
+    promptConfirm('isStoryPublic', 'Generate public story? (Selecting "No" will create an internal story)'),
+    promptInput(
+      'group',
+      'Select component group for the story (e.g. Forms, Layout)',
+      true,
+      'General',
+      ({ hasStory }) => hasStory
+    ),
```

Comment:
Would be nice if this was a select :)

Reply:
I was thinking the same but then you wanted to add a story for a new group, you'd have to manually modify the lost, altho not sure how often that would be used.